rpi0w-nexmon.sh

```diff
@@ -174,9 +160,11 @@ sed -i -e 's/^#PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/
 rm -f /etc/ssh/ssh_host_*_key*
 
 systemctl enable regenerate_ssh_host_keys
-
 update-rc.d ssh enable
 
+# Turn off kernel dmesg showing up in console since rpi0 only uses console
+echo "sudo dmesg -D" > rc.local
```

Comment:
sudo isn't needed here.  Also rc.local should have "exit 0" as a last line in case there are any issues and it doesn't block the system from booting completely.